eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Serialisation/GoldenUtils.hs

```diff
@@ -25,11 +25,11 @@ import Cardano.Binary
     toCBOR,
   )
 import Cardano.Ledger.Serialization (ToCBORGroup (..))
-import Cardano.Prelude (LByteString)
 import Codec.CBOR.Encoding (Encoding (..), Tokens (..))
 import Codec.CBOR.Term (decodeTerm)
 import Control.Exception (throwIO)
 import Control.Monad (unless)
+import qualified Data.ByteString.Lazy as Lazy (ByteString)
```

Comment:
For consistency sake let's use `as BSL`. In fact if you can grep for `as Lazy` and change the few occurrences there to `as BSL` too that would be great.
```suggestion
import qualified Data.ByteString.Lazy as BSL (ByteString)
```